framework/doc/content/source/fvkernels/FVDiffusion.md

```diff
@@ -18,10 +18,21 @@ order derivative term into a first order derivative integrated over a surface.
 
 where $\vec{n_f}$ is the surface normal on each side of the element considered.
 
-!alert note
-Unless specified otherwise using the `force_boundary_execution` or `boundaries_to_force`
-parameter, this kernel is not executed on variable boundaries. Unless a boundary condition is
-specified instead, this is equivalent to specifying a zero diffusive flux through the boundary.
+The diffusion coefficient can be interpolated to the surface using two approaches:
+
+- +Simple arithmetic average:+ $D_f = w_1 D_1 + (1-w_1) D_2$ (with $D_1$, $D_2$ being the diffusion
+  coefficient in the neighboring cells respectively)
+- +Simple harmonic average:+ $D_f = \frac{1}{\frac{w_1}{D_1} + \frac{1 - w_1}{D_2}}$, which yields better results
+  if the diffusion coefficients are positive and discontinuous. This is due to the fact that this scheme preserves
+  flux continuity in the face-normal direction on orthogonal grids.
+
+The interpolation method can be set using the [!param](/FVKernels/FVDiffusion/coeff_interp_method) parameter,
+and is defaulted to `harmonic` due to its superior accuracy for discontinuous diffusion coefficients.
+Simple tests cases with discontinuous diffusion coefficients (see below)
+indicate that using harmonic interpolation yields a second-order accurate
+scheme for orthogonal and 1D meshes and close to second-order (~1.8) accurate scheme for slightly
```

Comment:
this 1.8 is coincidental, dont mention it
close to second order seems good enough